CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Fixed
 * When a public name is defined on a property, calling `realm::Results::sort()` or `realm::Results::distinct()` with the internal name could throw an error like `Cannot sort on key path 'NAME': property 'PersonObject.NAME' does not exist`. ([realm/realm-js#6779](https://github.com/realm/realm-js/issues/6779), since v12.12.0)
+* Fixed sending empty UPLOAD messages when there is no download to acknowledge ([#2129](https://jira.mongodb.org/browse/RCORE-2129), since v14.8.0).
```

Comment:
Is this a correctness issue or a performance improvement? I don't think it is clear to someone just reading this line what the impact was.

Reply:
It's a performance improvement. It saves roundtrips to the server. I'll reword it.